ui/app/components/app/alerts/alerts.scss

```
@@ -0,0 +1 @@
+@import './unconnected-account-alert/unconnected-account-alert.scss'
```

Nit: missing semi-colon at the end of this line

```suggestion
@import './unconnected-account-alert/unconnected-account-alert.scss';
```